pypit/core/arskysub.py

```diff
@@ -118,13 +119,18 @@ def bg_subtraction_slit(slit, slitpix, edge_mask, sciframe, varframe, tilts,
     bgframe[all_slit] = skyset.value(piximg[all_slit])[0] #, skyset)
 
     # Debugging/checking
-    if False:
+    if PLOT_FIT:
         from matplotlib import pyplot as plt
+        from IPython import embed
```

Comment:
IPython is not included in our dependencies..  Remove

Reply:
My bad.